src/filters/annotation.js

```diff
@@ -24,6 +24,7 @@ angular.module('openshiftCommonUI')
       "jenkinsBuildURL":          ["openshift.io/jenkins-build-uri"],
       "jenkinsLogURL":            ["openshift.io/jenkins-log-url"],
       "jenkinsStatus":            ["openshift.io/jenkins-status-json"],
+      "loggingVisualURL":         ["openshift.io/logging.ui.url"],
```

Comment:
I'd prefer `loggingUIURL` to match the annotation (although I think `openshift.io/logging.ui.hostname` and `loggingUIHostname` would be more accurate, see my comment in the other PR).

You'll need to run `grunt build` to create the dist files

Reply:
See https://github.com/openshift/origin-web-common#contributing for how to run set up grunt